packages/expo-image/src/web/ImageWrapper.tsx

```diff
@@ -125,7 +125,10 @@ const ImageWrapper = React.forwardRef(
         // eslint-disable-next-line react/no-unknown-property
         fetchpriority={getFetchPriorityFromImagePriority(priority || 'normal')}
         onLoad={(event) => {
-          events?.onLoad?.forEach((e) => e?.(event));
+          // this ensures the animation will run, since the starting class is applied at least 1 frame before the target class set in the onLoad event callback
+          window.requestAnimationFrame(() => {
```

Comment:
wait, SSR will call onLoad events from images? thought it will just bake the <img> tags into the initial html...

Anyways, are you testing SSR on bare expo? any quick guide on how to test out the SRR in action?